lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -173,6 +173,7 @@ WasmBytecodeGenerator::GenerateFunction()
     m_currentFunc->wasmInfo = m_funcInfo;
     m_nestedIfLevel = 0;
     m_maxArgOutDepth = 0;
+    m_evalStack.Clear();
```

Comment:
Hmm, I'm thinking about how this is going to work out. If the scopes are properly formed and we ExitEvalStackScope all the way, m_evalStack should be empty anyway. If we didn't do that, would be we leaving the possibility of unreleased locations? But we really shouldn't be in the situation, right? Perhaps this is taken care of elsewhere.